### PR TITLE
endpoint to slurp up rss feeds

### PR DESCRIPTION
**Pull request by johnlucia**

### CODE REVIEW

File: app/lib/rss_feed.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class RssFeed
+
+  def initialize(feed_url)
+    @feed_url = feed_url;
+  end
+
+  def as_json
+    feed_url = URI.encode(@feed_url)
+    feed_body = HTTParty.get(feed_url).parsed_response
+    results = feed_body.is_a?(Hash) ? feed_body : Hash.from_xml(feed_body)
```
Comment (jcstringer) [inline app/lib/rss_feed.rb]: Would this ever actually be a ruby hash? Seems like it would always be either json or xml?

Comment (jcstringer) [reply app/lib/rss_feed.rb]: Oh maybe it's coming back as a Ruby object by way of HTTParty#parsed_response, that would make more sense.

File: spec/lib/rss_feed_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                          </rss> ' }
+
+  subject { described_class.new(feed_url).as_json }
+
+  before do
+    allow(HTTParty).to receive(:get).with(feed_url) { http_response }
+    allow(http_response).to receive(:parsed_response) { response_body }
+  end
+
+  it "passes on the resulting xml as json" do
```
Comment (jcstringer) [inline spec/lib/rss_feed_spec.rb]: Prob want a spec for the scenario where it's a Ruby object as well as the xml case.

**Comment by Matt-Bishop** [comment]
@johnlucia  - IF you got to send this thing through testing anyway - would you mind bumping these gems that have security alerts for this app.. - https://github.com/G5/g5-social-feed-service/network/alerts

**Pull request merged**